src/certificates/openssl.ts

```diff
@@ -5,9 +5,9 @@ import fs from "fs";
 
 const keyLength = 2048;
 
-async function isDHParamValid() {
+async function isDHParamValid(dhparamPath: string) {
```

Comment:
So either we have all functions have param path as a string, or we load all of them from config for. Its fine either way, but lets make it consistent.

Reply:
You're right. Done